### PR TITLE
Add captions and misc improvements in Ch8

### PR DESCRIPTION
**Pull request by jni**
@stefanv I remember some problem with formatting double spaces in markdown... I don't remember the resolution...

### CODE REVIEW

File: markdown/ch8.markdown [hunk 1]
```diff
@@ -372,6 +372,7 @@ def integer_histogram(counts, normed=True, xlim=[], ylim=[],
 counts_arr = np.fromiter(counts.values(), dtype=int, count=len(counts))
 integer_histogram(counts_arr, xlim=(-1, 250))
 ```
+<!-- caption text="Histogram of k-mer counts" -->
```
Comment (hdashnow) [inline markdown/ch8.markdown]: Same caption at line 536?

Comment (jni) [inline markdown/ch8.markdown]: @hdashnow well spotted! done!

File: markdown/ch8.markdown [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Big Data in Little Laptop with Toolz
 
-> GRACIE: A knife? The guy's twelve feet tall!
+> GRACIE: A knife? The guy's twelve feet tall!  
 > JACK: Seven. Hey, don't worry, I think I can handle him.
 >
 > — Jack Burton, Big Trouble in Little China
```
Comment (jni) [inline markdown/ch8.markdown]: Here's the html output from `make html/ch8.html` for me:

```html
<blockquote><p>GRACIE: A knife? The guy's twelve feet tall!<br/>
JACK: Seven. Hey, don't worry, I think I can handle him.</p>
<p>— Jack Burton, Big Trouble in Little China</p>
</blockquote>
```

And the wc3 page on `br`:

> Differences Between HTML and XHTML
> In HTML, the `<br>` tag has no end tag.  
> In XHTML, the `<br>` tag must be properly closed, like this: `<br />`.

So, I think it's correctly interpreted by notedown, we might just need to change `br/` to `br`, depending on the HTMLBook requirements.

**Comment by hdashnow** [comment]
Shall I merge this then? :)

**Comment by hdashnow** [comment]
Oh, wait, still inconsistent italics in the quote.

**Comment by jni** [comment]
@hdashnow it's good again, it just needed to be rebased on master. =)

File: data/iris-target.csv [hunk 3]
```diff
@@ -0,0 +1,150 @@
+0.000000000000000000e+00
```
Comment (stefanv) [inline data/iris-target.csv]: What a crazy way to store labels!  Should we change these to ints?

**Pull request merged**